### PR TITLE
feat(lerna): upgrade lerna to v3.19.0

### PR DESCRIPTION
**Pull request by nellyk**
upgrade lerna to v3.19.o
eslint was upgraded to v6 thus the js changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,16 +9,32 @@
...
-    "eslint-config-amex": "^11.0.1",
+    "eslint": "^5.14.1",
+    "eslint-config-amex": "^10.0.0",
+    "holocron": "*",
+    "holocron-module-route": "*",
     "husky": "^3.1.0",
-    "jest": "^24.8.0",
-    "lerna": "^2.9.0"
+    "immutable": "*",
+    "jest": "^24.1.0",
```
Comment (Francois-Esquire) [inline package.json]: Was the `jest` downgrade intentional?

File: package.json [hunk 2]
```diff
@@ -9,16 +9,32 @@
     "posttest": "npm run test:lint && npm run test:git-history"
   },
   "devDependencies": {
```
Comment (JamesSingleton) [inline package.json]: What is the reason for moving all the `devDependencies` into the parent `package-lock.json`

**Comment by nellyk** [comment]
@JamesSingleton as part of upgrading to lerna 3 the dev dependencies can be shared from the root across the project allow you to maintain them from a single package.json check this out https://github.com/lerna/lerna#common-devdependencies

File: package.json [hunk 3]
```diff
@@ -50,5 +66,10 @@
       "pre-commit": "npm test",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "dependencies": {
+    "@americanexpress/eslint-plugin-one-app": "6.0.0",
```
Comment (10xLaCroixDrinker) [inline package.json]: why are these needed?

**Comment by JAdshead** [comment]
Thanks Nelly for updating this for me. 👍

File: package.json (hunk 3 above)
Comment (nellyk) [reply package.json]: thank you for catching this, i've removed them

File: packages/one-app-bundler/package.json [hunk 4]
```diff
@@ -75,24 +75,6 @@
     "webpack-subresource-integrity": "^1.3.4",
     "yargs": "^12.0.5"
   },
-  "devDependencies": {
-    "@americanexpress/one-app-ducks": "*",
-    "@americanexpress/one-app-router": "*",
-    "core-js": "^3.2.1",
-    "holocron": "*",
```
Comment (anescobar1991) [inline packages/one-app-bundler/package.json]: Are these dev deps used in any other package in this monorepo? I thought the practice was to only put the "common" dev deps in the top level package.json file but these devDeps are only used in this package I thought...

Comment (nellyk) [reply packages/one-app-bundler/package.json]: Thank you for the feedback, i'll update this to have only shared dev dependencies.
https://github.com/lerna/lerna#common-devdependencies

Comment (nellyk) [reply packages/one-app-bundler/package.json]: updated on 6b18512380bbfb1cff491b821b10dbc037e4d1a9
